composer/trainer/ddp.py

```diff
@@ -10,6 +12,8 @@
 from composer.utils import dist
 from composer.utils.string_enum import StringEnum
 
+__all__ = ["DDPSyncStrategy", "ddp_sync_context", "prepare_ddp_module"]
+
 
 class DDPSyncStrategy(StringEnum):
```

Comment:
Thoughts about documenting this on the module level page for the trainer? Then, we can make this module private.

Reply:
Resolved offline